ant/windows/windows-packager.nsi.in

```diff
@@ -178,16 +174,20 @@ SectionEnd
 ;-------------------------------
 
 Function .onInit
+  ${If} ${RunningX64}
+      SetRegView 64
+  ${EndIf}
+  ${If} $InstDir == ""  
     ${If} ${RunningX64}
-        SetRegView 64
-        StrCpy $INSTDIR "$PROGRAMFILES64\${socket.name}"
+      StrCpy $INSTDIR "$PROGRAMFILES64\${socket.name}"
+    ${Else}
+      StrCpy $INSTDIR "$PROGRAMFILES\${socket.name}"
     ${EndIf}
+  ${EndIf}
 FunctionEnd
 
 Function un.onInit
     ${If} ${RunningX64}
         SetRegView 64
-        StrCpy $INSTDIR "$PROGRAMFILES64\${socket.name}"
```

Comment:
So this 64-bit directory logic isn't needed here?  That's good to know if that is the case.  I assume it's because the uninstaller already knows where it is installed?  I assume the `EnableX64FSRedirection` handles the Wow64 and x86 problems, right, so we're covered on all bases?